ratelimitj-hazelcast/src/main/java/es/moki/ratelimitj/hazelcast/HazelcastSlidingWindowRequestRateLimiter.java

```diff
@@ -92,7 +91,7 @@ public boolean resetLimit(String key) {
         return hz.getMap(key);
     }
 
-    private boolean eqOrGeLimit(String key, int weight, boolean strictlyGreater) {
+    private boolean eqOrGeLimit(String key, int weight, boolean strictlyGreater, boolean increment_anyway) {
```

Comment:
`increment_anyway` -> `incrementAnyway`

For me `alwaysIncrement` sounds better.